src/standard/utils.html

```diff
@@ -363,28 +363,47 @@
      * @return {HTMLLinkElement} The link element for the URL to be loaded.
      */
     importHref: function(href, onload, onerror, optAsync) {
-      var l = document.createElement('link');
-      l.rel = 'import';
-      l.href = href;
-
-      optAsync = Boolean(optAsync);
-      if (optAsync) {
-        l.setAttribute('async', '');
-      }
-
+      var link = document.createElement('link');
+      link.rel = 'import';
+      link.href = href;
+      var list = Polymer.Base.importHref.imported = 
+        Polymer.Base.importHref.imported || {};
+      var cached = list[link.href];
+      var imprt = cached || link;
       var self = this;
       if (onload) {
-        l.onload = function(e) {
+        var loadListener = function(e) {
+          e.target.__firedLoad = true;
+          e.target.removeEventListener('load', loadListener);
           return onload.call(self, e);
         }
+        imprt.addEventListener('load', loadListener);
       }
       if (onerror) {
-        l.onerror = function(e) {
+        var errorListener = function(e) {
+          e.target.__firedError = true;
+          e.target.removeEventListener('error', errorListener);
           return onerror.call(self, e);
         }
```

Comment:
No, import can be cached element if present `var imprt = cached || link;`. And below load/error events will be simulated on cached element if it is present or link will be loaded if necessary. So in either case `imprt` is what events will be fired on.

However, I do not understand the purpose of shortening `import` to `imprt` :smile: 
Also link creation might be avoided if cached import is already in place, this will save few wasted CPU cycles.